### ISSUE TITLE
No Itemid in link

### ISSUE THREAD

**Issue by pinta83** [getk2/k2-links-for-jce#9]
This pugin does not add "&Itemid=xxx" if there is a link to the items category in the menu.

If there is no Itemid, the sef link is generated for example
/component/k2/item/article-alias.html

If you have and Itemid in the non seo link the link is generated like
/menu-alias-of-linked-category/item/article-alias.html

There are a few Advanced link K2 plugins for JCE out there, but yours does not use Itemid, and the other ones use it, but they dont have article alias in the link...

So basically there is no K2 ADV Link plugin that does the job 100% right...

**Comment by lamberticus** [comment]
As far as I know this issue is still not fixed, I was experiencing the same issue when I upgraded to JCE 2.6 and installed the K2 links plugin. I did find some info that deals with this issue:

https://www.aftercube.com/tweaks-and-tricks/4-k2-links-for-jce-v2-6-0-plugin

The line numbers in the most recent version file are not the same as listed here (163 vs 181) but the fix does work.

**Comment by fevangelou** [comment]
Resolved in v2.6.1 now available.
